trade-finance-manager-api/src/v1/helpers/amendment.helpers.js

```diff
@@ -311,6 +311,30 @@ const sendFirstTaskEmail = async (taskVariables) => {
   }
 };
 
+/**
+ * Initiate an internal UKEF team email, upon an amendment
+ * submission (automatic) and approval (manual)
+ * @param {String} ukefFacilityId UKEF Facility ID
+ */
+const internalAmendmentEmail = async (ukefFacilityId) => {
+  try {
+    const templateId = EMAIL_TEMPLATE_IDS.INTERNAL_AMENDMENT_NOTIFICATION;
+    const sendToEmailAddress = process.env.UKEF_INTERNAL_NOTIFICATION;
+    const emailVariables = {
+      ukefFacilityId,
+    };
+
+    if (!templateId || !sendToEmailAddress || !ukefFacilityId) {
+      throw new Error('Void argument specified');
```

Comment:
do we need this? it should be captured by the `try` block if the `sendTfmEmail` fails

Reply:
I have replaced with return `false`.
GovNotify API will return appropriate response, but there is no check in place to verify whether the `emailVariable` object is valid or not. I think need a check in place to avoid invoking API if `ukefFacilityId` is null.